gandalf.js

```diff
@@ -1,17 +1,18 @@
-/**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?